models/intel/text-spotting-0002/description/text-spotting-0002.md

```diff
@@ -1,42 +1,49 @@
-# text-spotting-0002-detector
+# Composite model name: text-spotting-0002
 
 ## Use Case and High-Level Description
 
-This is a text spotting model that simultaneously detects and
+This is a text spotting composite model that simultaneously detects and
 recognizes text. The model detects symbol sequences separated by space and performs
 recognition without a dictionary. The model is built on top of the Mask-RCNN
 framework with additional attention-based text recognition head.
 
 Symbols set is alphanumeric: `0123456789abcdefghijklmnopqrstuvwxyz`.
 
-This model is a Mask-RCNN-based text detector with ResNet50 backbone and additional text features output.
+*Hmean Word spotting* is defined and measured according to the
+[Incidental Scene Text (ICDAR2015) challenge](https://rrc.cvc.uab.es/?ch=4&com=introduction).
```

Comment:
This seems like it should be in the "Composite model specification" section, not here.

Reply:
Done